### DISCUSSION TITLE
Warnings saying “algorithm did not converge”.

### DISCUSSION THREAD

**Discussion by anamgreco** [amices/mice#357] [Q&A]
Dear all,

First of all, an enormous thank you to @stefvanbuuren and collaborators! It is my first time posting here so apologies if I am not doing it correctly. 

I am encountering some difficulties when using multiple imputed datasets, so thank you in advance for your help:

1-	I am running mice in a dataset with around 5% of missing data. I have 42 variables, but only 2 of them are continuous, the rest are all categorical (with two or more categories). I am using the following code, specifying the method for each variable in meth:
`# Perform multiple imputation:
imputed_data <- mice::mice(df, m = 10, method = meth, 
                           maxit = 10, seed = 12345, print = FALSE)
`
I checked with plots all the imputed datasets and everything looks fine, but I get around 21 warnings saying “algorithm did not converge”. I am not sure if I should do something differently…

THANK YOU!

**Comment by stefvanbuuren** [comment]
Thanks. Please restrict your issues to one point, not multiple points. 

I have changed the title to reflect your first question so that other with the same warning can find it.

The warning is likely to come from the categorical imputation routines. You get more of those warnings of the proportions are closer to zero or one. If the graphs look OK, there's no need to worry.

**Comment by anamgreco** [comment]
Thank you very much, @stefvanbuuren ! I will now edit the post to address only one point per issue. Apologies.
